.github/actions/init_rust_job/action.yml

```diff
@@ -33,24 +33,24 @@ runs:
 
     - name: Extract the Rust version to use from the `rust-toolchain.toml` file
       shell: bash
+      working-directory: cli
       run: |
-        cd cli
         rust_version=$(grep "channel" rust-toolchain.toml | cut -d "=" -f 2 | cut -d " " -f 2)
         echo "RUST_VERSION=$rust_version" >> $GITHUB_ENV
 
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@master
+      uses: actions-rust-lang/setup-rust-toolchain@v1
       with:
         toolchain: ${{ env.RUST_VERSION }}
-        components: clippy, rustfmt
+        components: clippy,rustfmt
+        cache: 'false'
 
     - name: rustup
       shell: bash
-      run: |
-        rustup target list
+      run: rustup target list
 
     - name: Install cargo-nextest & libs linux
-      if: ${{ startsWith(inputs.platform, 'linux') }}
+      if: ${{ inputs.platform == 'linux' }}
```

Comment:
Does not work for linux-arm

Reply:
The `platform` input field is a "choice" input, and it can only be one of `windows`, `macos` and `linux`.

Reply:
Turns out, we are both correct. The choices are limited to those three, but that doesn't apparently mean you can't also use other values for the input. I'll update the code, and add `linux-arm` as another valid choice 👍

Reply:
Resolved in [63e3f7c](https://github.com/grafbase/grafbase/pull/611/commits/63e3f7c48186a96422b2a61f7f7ba00ed8d653b6).